remoteworkitem/tracker_repository_test.go

```diff
@@ -55,6 +58,33 @@ func (test *TestTrackerRepository) TestTrackerCreate() {
 	assert.NotNil(t, tracker2)
 }
 
+func (test *TestTrackerRepository) TestExistsTracker() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+
+	t.Run("tracker exists", func(t *testing.T) {
+		t.Parallel()
+		// given
+		tracker, err := test.repo.Create(context.Background(), "http://api.github.com", ProviderGithub)
+		assert.Nil(t, err)
```

Comment:
Fixed in https://github.com/almighty/almighty-core/pull/1389/commits/960ba3deddb447e3b51d11c251d1ee00c34799c3